lib/shipshape/actions/aws/parameter_store.rb

```diff
@@ -46,11 +47,40 @@ def get_all_parameters(path)
         end
 
         def put_opts
-          default = { overwrite: true, type: 'String' }
+          return default_opts.merge(encrypt_opts) if options['encrypt']
 
-          return default.merge(key_id: options['kms_key_id'], type: 'SecureString') if options['encrypt']
+          default_opts
+        end
+
+        def default_opts
+          {
+            overwrite: true,
+            type: 'String'
+          }
+        end
 
-          default
+        def encrypt_opts
+          {
+            key_id: options['kms_key_id'],
+            type: 'SecureString'
+          }
+        end
+
+        def parameter_tags(resource_id, prefix)
+          {
+            resource_type: "Parameter",
+            resource_id: resource_id,
+            tags: [
+              {
+                key: prefix.app_env,
```

Comment:
the key/vals are reversed because the console only allows filtering by the key for some reason.